src/autoyast-rnc/kdump.rnc

```diff
@@ -6,11 +6,14 @@ namespace config = "http://www.suse.com/1.0/configns"
 kdump =
   element kdump {
      (kdump_crash_kernel_value | kdump_crash_kernel_list)? &
+    # parameter specify crashkernel value for xen hypervisor
+     kdump_crash_xen_value? &
```

Comment:
This needs to mimic the above line because Export uses either an array or a single string too.